src/services/myAccount.service.js

```diff
@@ -60,8 +58,30 @@ const getMyLeadListings = async (req = {}) => {
     params.push(endRange);
   }
 
-  const myCars = await query(carQuery, params);
-  return myCars;
+  if (isSub && filterValue === 'ALL') {
+    const q = `${carQuery}`;
+    myCars.leads = await query(q, params);
+    return myCars;
+  }
+
+  if (isSub && filterValue !== 'ALL') {
+    const q = `${carQuery} ORDER BY leads.id`;
+    myCars.leads = await query(q, params);
+    return myCars;
+  }
+
+  if (!isSub && filterValue === 'ALL') {
+    const q = `${carQuery} ORDER BY leads.id LIMIT 10`;
+    const allLeads = await query(q, params);
+    const completeLeads = allLeads.slice(0, 5);
+    const partialLeads = allLeads.slice(5, 10).map((lead) => {
+      return { id: lead.id, name: lead.name, city: lead.city, make: lead.make, model: lead.model };
+    });
+
```

Comment:
This is fine for `!isSub`.
But If user is a premium and `isSub` is true, then
the results should be paginated.